File: client/src/scripts/classes/driver.js
const getTime = () => {
  const now = new Date();
  const time = now.toISOString().toString();
  return time;
};

class Driver {
  constructor() {
    this.edit = false;
    this.dateHired = getTime();
  }
  static deserialize(data) {
    const driver = new Driver();
    driver.id = data.id;
    driver.name = data.name;
    driver.rating = data.rating;
    driver.dateHired = data.dateHired;
    driver.deleted = data.deleted;
    return driver;
  }
}

File: api/Models/Driver.cs
namespace api.Models
{
    public class Driver
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public int Rating { get; set; }
        public DateTime DateHired { get; set; }
        public bool Deleted { get; set; }

        public string ToString()
        {
            return "Driver: " + Name + " Rating: " + Rating + " Date Hired: " + DateHired + " Deleted: " + Deleted;
        }
    }

}